testsuite/grpc-tests/pom.xml

```diff
@@ -337,33 +358,33 @@
                     </execution>
                 </executions>
             </plugin>
-
             <plugin>
                 <groupId>org.codehaus.mojo</groupId>
                 <artifactId>exec-maven-plugin</artifactId>
-                <version>3.0.0</version>
                 <executions>
                     <execution>
                         <id>genproto</id>
-                        <phase>generate-sources</phase>
+                        <phase>process-sources</phase>
                         <goals>
                             <goal>java</goal>
                         </goals>
                         <configuration>
                             <mainClass>dev.resteasy.grpc.bridge.generator.protobuf.JavaToProtobufGenerator</mainClass>
                             <cleanupDaemonThreads>false</cleanupDaemonThreads>
-                            <arguments>
-                                <argument>${project.basedir}</argument>
-                                <argument>dev.resteasy.grpc.example</argument>
-                                <argument>dev.resteasy.grpc.example</argument>
-                                <argument>${root.class}</argument>
-                            </arguments>
+                            <!--additionalClasspathElements>${project.build.directory}/classes</additionalClasspathElements-->
+                            <commandlineArgs>${project.basedir} dev.resteasy.grpc.example dev.resteasy.grpc.example ${root.class} -D"--add-opens java.base/java.util=ALL-UNNAMED"</commandlineArgs>
```

Comment:
This seems to work, but it's not valid XML. The " should be changed to `&quot;`.

Reply:
Fixed